src/services/pathContents/pull/dir/constants.ts

```diff
@@ -0,0 +1,148 @@
+import { z } from 'zod'
+
+import { MissingHeadReportSchema } from 'services/comparison'
+import { UnknownFlagsSchema } from 'services/impactedFiles/schemas'
+
+const MissingCoverageSchema = z.object({
+  __typename: z.literal('MissingCoverage'),
+  message: z.string(),
+})
+
+const UnknownPathSchema = z.object({
+  __typename: z.literal('UnknownPath'),
+  message: z.string(),
```

Comment:
Would recommend adding `nullish` here. had some issues locally where there was no message being sent with the error types.